cocotb/share/makefiles/simulators/Makefile.vcs

```diff
@@ -55,6 +55,12 @@ ifeq ($(ARCH),x86_64)
 EXTRA_ARGS += -full64
 endif
 
+ifeq ($(GUI),1)
+    EXTRA_ARGS += -gui
+else
+    EXTRA_ARGS +=
+endif
```

Comment:
You don't need the else part, please remove it.

Reply:
Done.